properties.schema

```diff
@@ -258,61 +258,71 @@
             "title": "Menu item background colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Sets the background colour for the Menu Item."
           },
           "menu-item-inverted": {
             "title": "Menu item foreground colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Defines the Menu Item text colour. Should be a colour that provides good contrast against the Menu Item colour."
           },
           "menu-item-border-color": {
             "title": "Menu item border colour",
             "type": "string",
             "inputType": "ColourPicker",
-            "default": ""
+            "default": "",
+            "help": "Defines the border colour that appears around the Menu Item."
```

Comment:
I'm not sure this help text really adds anything, the title seems pretty self-explanatory. Seems minor I know but it would save on i18n effort if left out.